build-scripts/cpe_generate.py

```diff
@@ -29,7 +30,12 @@ def parse_args():
         "definitions and the tests, states objects and variables it "
         "references, and then write them into a standalone OVAL CPE file, "
         "along with a synchronized CPE dictionary file.")
-    p.add_argument("product", help="Name of the product")
+    p.add_argument(
+        "--product-yaml",
+        help="YAML file with information about the product we are building. "
+        "e.g.: ~/scap-security-guide/rhel7/product.yml "
+        "needed for autodetection of profile root"
+    )
```

Comment:
Not sure how much of a point there is, but for consistency it might be nice to pass in build config as well and use `open_environment` below and call it `env_yaml`. My 2c.

Reply:
Yeah I'm fine leaving this. Just thought I'd mention. :)